### ISSUE TITLE
Not able to write snapshots based tests due to dynamic id generation

### ISSUE THREAD

**Issue by pranshuchittora** [twilio-labs/paste#2845]
### Description

Hey Folks, 
I am trying to write some snapshots based UTs using react testing library. But unable to due to dynamic id generation

<img width="896" alt="Screenshot 2022-11-23 at 17 46 50" src="https://user-images.githubusercontent.com/32242596/203545037-ee29e2ee-09a9-466b-a8b9-e80881856080.png">

I tried wrapping the component with a theme provider, but still the its are dynamic. Let me know if I am doing anything incorrectly or there's any workaround for the same

### Link to Reproduction

NA

### Paste Core Version

10.20.0

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by SiTaggart** [comment]
Hi @pranshuchittora 

This functionality is working as intended for accessibility reasons. 

I go into some detail about the why in this discussion post https://github.com/twilio-labs/paste/discussions/1940

TL;DR

Don’t write snapshot tests. A better more reliable and intentional approach to testing is to expect specific functionality, or attributes to be found in the DOM. A failed snapshot test doesn’t tell you if the functionality or expected behavior failed, just that some thing changed.
